<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoLyricsCreator</class>
 <widget class="QMainWindow" name="VideoLyricsCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频字幕制作</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>B站视频链接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="urlEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="cidLabel">
          <property name="text">
           <string>CID</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>起始时间偏移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="offsetSpin">
            <property name="suffix">
             <string>毫秒</string>
            </property>
            <property name="maximum">
             <number>6000000</number>
            </property>
            <property name="singleStep">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="text">
           <string>开始发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>歌词字幕</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="lyricsEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>账号</string>
    </property>
    <addaction name="actionSet_Cookie"/>
    <addaction name="actionSet_Lyrics_Danmaku_Sample"/>
    <addaction name="actionSet_Sample_Help"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_Lyrics_Danmaku_Sample">
   <property name="text">
    <string>设置字幕样例</string>
   </property>
  </action>
  <action name="actionSet_Sample_Help">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionSet_Cookie">
   <property name="text">
    <string>设置Cookie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
